frontend/src/services/service-docs.http-service.tsx

```diff
@@ -0,0 +1,98 @@
+import { ServiceDocsApi } from 'msadoc-client';
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import { firstValueFrom } from 'rxjs';
+
+import {
+  ListAllServiceDocsHttpResponse,
+  UnknownHttpError,
+} from '../models/api';
+import { APP_ROUTES } from '../routes';
+
+import { useAuthDataServiceContext } from './auth-data-service';
+import { useHttpServiceContext } from './http-service';
+
+interface ServiceDocsHttpService {
+  listAllServiceDocs: () => Promise<
+    ListAllServiceDocsHttpResponse | UnknownHttpError
+  >;
+}
+function useServiceDocsHttpService(): ServiceDocsHttpService {
+  const navigate = useNavigate();
+  const authDataService = useAuthDataServiceContext();
+  const httpService = useHttpServiceContext();
+
+  /**
+   * @param accessToken The Access Token to use when performing the request. If no token is specified, the one provided by the AuthDataService is used.
+   */
+  async function listAllServiceDocs(
+    accessToken?: string,
+  ): Promise<ListAllServiceDocsHttpResponse | UnknownHttpError> {
```

Comment:
I think we don't need the `accessToken` parameter anymore, right? I have originally introduced this to make token refreshing (on 401) easier.

Reply:
Addressed in [1025a20](https://github.com/osrgroup/msadoc/pull/91/commits/1025a208041454539d84579e76865de36a9e6521)